src/status_im/contexts/wallet/edit_account/view.cljs

```diff
@@ -33,7 +33,8 @@
                   {:account    edited-account-data
                    :on-success #(show-save-account-toast updated-key)}])))
 
-(def view
+(defn view
+  []
   (let [edited-account-name  (reagent/atom nil)
         show-confirm-button? (reagent/atom false)
```

Comment:
Incredible that this happened and it worked.

I see the problem as atoms were permanent and didn't restared its state.

Really nice catch @smohamedjavid !

Reply:
Not going to lie, took a while to figure out why the atom didn't reset 😅. 


<img width="300" src="https://github.com/status-im/status-mobile/assets/19339952/a3101472-3f4b-4ae7-aefb-a874c478611f" />